hbase-procedure/src/main/java/org/apache/hadoop/hbase/procedure2/ProcedureScheduler.java

```diff
@@ -95,6 +95,20 @@
    */
   boolean hasRunnables();
 
+  /**
+   * Fetch one high priority Procedure from the queue
+   * @return the Procedure to execute, or null if nothing present.
+   */
+  Procedure pollHighPriority();
+
+  /**
+   * Fetch one high priority Procedure from the queue
+   * @param timeout how long to wait before giving up, in units of unit
+   * @param unit a TimeUnit determining how to interpret the timeout parameter
+   * @return the Procedure to execute, or null if nothing present.
+   */
+  Procedure pollHighPriority(long timeout, TimeUnit unit);
```

Comment:
What is high priority? We have priority on rpcs with a gradient where 200 is high priority. Here there only gradient is high or not high? And only the assign of meta is high priority?

Reply:
high priority procedure is meta procedure/ServerCrashProcedure(carry meta)